litellm/proxy/proxy_server.py

```diff
@@ -2604,6 +2604,9 @@ async def startup_event():
     ### CHECK IF VIEW EXISTS ###
     if prisma_client is not None:
         create_view_response = await prisma_client.check_view_exists()
+        # Apply misc fixes on DB
+        # [non-blocking] helper to apply fixes from older litellm versions
+        asyncio.create_task(prisma_client.apply_db_fixes())
```

Comment:
hey @ishaan-jaff this doesn't look like a one-time check.

I'm seeing it on my logs every time i start the proxy w/ db.

Reply:
hmm - can you delete the check @krrishdholakia ? 

- I believe this issue should be resolved for all our users  / the ones impacted by the original error